vars/withGo.groovy

```diff
@@ -23,6 +23,10 @@ def call(Map parameters = [:], body) {
 
     def alwaysPullImage = goImage.endsWith(":latest")
 
+    envVars.add(containerEnvVar(key: 'LD_PRELOAD',value: 'libnss_wrapper.so'))
```

Comment:
I think that this is no the place to add those environment variables.
They should instead be added to the Jenkinsfile(s) that are actually using `withGo`.

Now, if they are widely used and we want to avoid the boilerplate, I'd add a new function called  `withNssWrapper` that would define an empty podTemplate with just these env vars ass `podEnvars` This would allow them to be copied to all container of the pod.